api-spec/WFS3core+STAC.yaml

```diff
@@ -0,0 +1,631 @@
+openapi: 3.0.1
+info:
+  title: STAC API conforming to the OGC Web Feature Service standard
+  version: 0.0.1
+  description: >-
+    This is a sample OpenAPI definition that conforms to the OGC Web Feature
+    Service specification (conformance classes: "Core", "GeoJSON", "HTML" and
+    "OpenAPI 3.0").

```

Comment:
Would be good to add in the fact that this also implements the STAC specification. Indeed the description should be more like 'This is a sample OpenAPI definition of a SpatioTemporal Asset Catalog API that also conforms to the OGC ...'